FBSDKCoreKit/FBSDKCoreKit/Internal/FBSDKImageDownloader.m

```diff
@@ -38,9 +38,21 @@ + (FBSDKImageDownloader *)sharedInstance
 - (instancetype)init
 {
   if ((self = [super init])) {
+#if TARGET_OS_MACCATALYST
```

Comment:
I think you need an && clause  here to pass the travis build. Haven't run it but will probably be something like this.
```suggestion
#if TARGET_OS_MACCATALYST && @available(iOS 13.0, *)
```

Reply:
That's not valid syntax in Objective-C. Also, I noticed that you decided to use the deprecated `... diskPath:` method on iOS, so you'll get a warning when people switch to iOS 13 using Xcode 11.